spec/chrome_integration/gtalk.spec.js

```diff
@@ -0,0 +1,237 @@
+/*
```

Comment:
Sorry, I meant to move the file from spec/integration to
spec/chrome_integration but accidentally made a duplicate.  This is fixed
now

On Thu Jan 22 2015 at 4:01:27 PM Raymond Cheng <notifications@github.com>
wrote:

> In spec/chrome_integration/gtalk.spec.js
> <https://github.com/freedomjs/freedom-social-xmpp/pull/85#discussion_r23409342>
> :
>
> > @@ -0,0 +1,237 @@
> > +/*
>
> Very confused. I think this same file is checked in twice?
> spec/integration/gtalk.spec.js
> spec/chrome_integration/gtalk.spec.js
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/freedomjs/freedom-social-xmpp/pull/85/files#r23409342>
> .
>